tutorials/fundamentals/concat-regexp.md

```diff
@@ -0,0 +1,14 @@
+To concatenate a regular expression in JavaScript, you can use a combination
+of the `+` operator, and the `new RegExp()` class:
+
+```javascript
+let reg = /mastering/g;
+let exp = /js/y;
+let flags = reg.flags+exp.flags;
+flags = Array.from(new Set(flags.split(''))).toString();
+let regexp = new RegExp(reg.source + exp.source, flags);
+```
+
+The reason why you should be removing duplicates from the flags is that
+when you put in a flag that is already present, the computer does not recognize it
```

Comment:
This sentence is too long, and "the computer" is not a good word choice here. Let's replace this with something like "You are responsible for removing duplicate flags. If you pass a duplicate flag to `new RegExp()`, JavaScript will throw a `SyntaxError: Invalid flags`."